### ISSUE TITLE
mvn clean install fails with Authentication failure on https://regnosys.jfrog.io

### ISSUE THREAD

**Issue by RandalP** [REGnosys/rosetta-code-generators#45]
### Description

 Running "mvn clean install" fails on a new clone of the database.

### Is this a regression?

 
 I assume so, but perhaps everyone else has a login to regnosys.jfrog.io ....

### Expected behaviour

 Following the build instructions in the ReadMe would work without error.

### Steps to reproduce
1. Clone repository
2. Run "mvn clean install"
3. Note failure during build of code-gen-daml
4. See error below

## 🔥 Exception or Error
<pre><code>
[ERROR] Failed to execute goal org.eclipse.xtend:xtend-maven-plugin:2.15.0:compile (Generate xtend sources) on project daml: Execution Generate xtend sources of goal org.eclipse.xtend:xtend-maven-plugin:2.15.0:compile failed: java.lang.IllegalStateException: Resource has not been loaded -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :daml
</code></pre>

After running mvn clean install -X, it contains numerous errors like the following:
<pre><code>

org.eclipse.aether.transfer.MetadataTransferException: Failure to transfer org.eclipse.emf:org.eclipse.emf.common/maven-metadata.xml from https://regnosys.jfrog.io/regnosys/libs-snapshot was cached in the local repository, resolution will not be reattempted until the update interval of regnosys-snapshots has elapsed or updates are forced. Original error: Could not transfer metadata org.eclipse.emf:org.eclipse.emf.common/maven-metadata.xml from/to regnosys-snapshots (https://regnosys.jfrog.io/regnosys/libs-snapshot): Authentication failed for https://regnosys.jfrog.io/regnosys/libs-snapshot/org/eclipse/emf/org.eclipse.emf.common/maven-metadata.xml 401 Unauthorized
</code></pre>

## 🌍  Your Environment

<pre><code>

Maven home: /usr/local/Cellar/maven/3.6.3_1/libexec
Java version: 14.0.1, vendor: Oracle Corporation, runtime: /Library/Java/JavaVirtualMachines/jdk-14.0.1.jdk/Contents/Home
OS name: "mac os x", version: "10.15.5", arch: "x86_64", family: "mac"
</code></pre>

**Anything else relevant?**

This is superficially similar to #32, but that is "Error code 403, Forbidden" rather than "401 Unauthorized"

**Comment by RandalP** [comment]
The problem that I reported is probably not so important after all. 

I eventually worked out that the real problem is that xtend-maven-plugin 2.15 was failing to compile the Java code with "Unsupported class file major version 58" errors, which corresponds to Java 14. Changing daml/pom.xml to target Java version 1.8 didn't make any difference.

Even recent versions of xtend-maven-plugin (2.22) only support Java 11, so I reverted to JDK 8, which resolved the build issue. This information should probably be added to the README.rst file.
